core/frontend/src/components/mavlink/MavlinkUpdater.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <span />
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { getModule } from 'vuex-module-decorators'
+
+import MavlinkStore from '@/store/mavlink'
+
+const mavlink_store: MavlinkStore = getModule(MavlinkStore)
+
+export default Vue.extend({
+  name: 'MavlinkUpdater',
+  async mounted() {
+    mavlink_store.setMessageRefreshRate({ messageName: 'HEARTBEAT', refreshRate: 1 })
+    mavlink_store.setMessageRefreshRate({ messageName: 'SYS_STATUS', refreshRate: 3 })
+  },
```

Comment:
Id `setMessageRefreshRate` async? If yes its missing an await here. If not there's no need for `mounted` to be async, I think.